platform-sdk/swirlds-cli/src/main/java/com/swirlds/cli/logging/HtmlGenerator.java

```diff
@@ -624,16 +694,31 @@ private static String generateFiltersDiv(
                 CLASS_NAME_COLUMN_LABEL,
                 REMAINDER_OF_LINE_COLUMN_LABEL)));
 
-        filterDivBuilder.append(createStandardFilterDiv(
-                "Log Level",
-                logLines.stream()
-                        .map(LogLine::getLogLevel)
-                        .distinct()
-                        .sorted(Comparator.comparing(Level::toLevel))
-                        .toList()));
-        filterDivBuilder.append(createStandardFilterDiv(
-                "Log Marker",
-                logLines.stream().map(LogLine::getMarker).distinct().toList()));
+        final Map<String, String> logLevelLabels = Map.of(
+                "TRACE", "trace-label",
+                "DEBUG", "debug-label",
+                "INFO", "info-label",
+                "WARN", "warn-label",
+                "ERROR", "error-label",
+                "FATAL", "fatal-label");
```

Comment:
Could this be a private static variable?

Reply:
done